src/Footer/Footer.js

```diff
@@ -0,0 +1,16 @@
+const Footer = () => {
+    return (
+            <footer id="footer" className="footer-1">
```

Comment:
@liamlwrnc 
Please can you talk me through why you have 5 divs?
Also, the wireframe shows the footer text inline with the content whereas this footer is not.